src/Microsoft.SymbolStore.Client/StoreQueryBuilder.cs

```diff
@@ -17,14 +19,29 @@ internal static class StoreQueryBuilder
 
         public static readonly string SourcePrefix = "sha1";
```

Comment:
The sha1 indexing scheme can be used for any purpose, it just happens that currently its being used for source. 

The prefix is also "sha1-"